dist/cleanup/index.js

```diff
@@ -23,13 +24,14 @@ async function cleanup() {
     // environment variables, so we overwrite the current value with an empty
     // string. The AWS CLI and AWS SDKs will behave correctly: they treat an
     // empty string value as if the environment variable does not exist.
-    core.exportVariable('AWS_ACCESS_KEY_ID', '');
-    core.exportVariable('AWS_SECRET_ACCESS_KEY', '');
-    core.exportVariable('AWS_SESSION_TOKEN', '');
-    core.exportVariable('AWS_DEFAULT_REGION', '');
-    core.exportVariable('AWS_REGION', '');
-  }
-  catch (error) {
+    core.exportVariable("AWS_ACCESS_KEY_ID", "");
+    core.exportVariable("AWS_SECRET_ACCESS_KEY", "");
+    core.exportVariable("AWS_SESSION_TOKEN", "");
+    core.exportVariable("AWS_DEFAULT_REGION", "");
+    core.exportVariable("AWS_REGION", "");
+
+    fs.rmSync(`${process.env.HOME}/.aws`, { force: true, recursive: true });
```

Comment:
@GeertWille Please recheck this line/imports
Looks like something went wrong, getting this error:
```
Post job cleanup.
Error: fs.rmSync is not a function
```